science/bedtools/Portfile

```diff
@@ -18,25 +19,22 @@ long_description    The BEDTools utilities allow one to address common genomics
                     The utilities are largely based on four widely-used file formats: \
                     BED, GFF/GTF, VCF, and SAM/BAM.
 
-checksums           rmd160  2a0fc10ce07f6228ee006e297577560bc24e35af \
-                    sha256  eb7028247d0e4732f113076fdbd580b8601365f7830cf19649d4d3116af08dfd
+checksums           rmd160  0807fc84acd6d80f646ee387943ff6345203396a \
+                    sha256  511d8623038836ac8d1f9e260016867494b9040de65d0f0f1eb7feac7b698bbd \
 
 depends_build       port:py27-sphinx
 
-depends_lib         port:zlib
+depends_lib         port:zlib port:lzma
```

Comment:
`depends_lib` should be changed to `depends_lib-append` to not overwrite portgroup dependencies.

Reply:
ok, done. thanks.